plugins/CoreAdminHome/templates/whatIsNew.twig

```diff
@@ -0,0 +1,24 @@
+<div piwik-content-intro>
+    <h2>{{ 'CoreAdminHome_WhatIsNewTitle'|translate|e('html_attr') }}</h2>
+</div>
+<div class="whatisnew">
+
+    {% for change in changes %}
+
+        <div class="card">
+            <div class="card-content">
+                <span style="float: left; font-size:32px; color:#3450A3; margin-right:10px" class="icon-new_releases"></span>
+                <h2 class="card-title">{{ change.plugin_name }} - {{ change.title }}</h2>
+                {{ change.description | raw }}
```

Comment:
is it on purpose to have that printed with `|raw`? In theory someone could place some eval code into the changes and it wouldn't be filtered. I'm not sure if we should allow any HTML within those change messages at all. Imho we don't need that. Or is that required @tsteur

Reply:
It's on purpose, the original issue screenshots show bullet points in the description and said `Description (allowing HTML)`. Could markdown be an option instead?

Reply:
Not sure here. We could also use `strip_tags` and only allow some tags like `<ul><li><br><b><strong><i><em>` and maybe `<img>`, as they would be blocked anyways, if they aren't local